app/views/charts/_simple_column.html.erb

```diff
@@ -0,0 +1,43 @@
+<script type="text/javascript">
+  am4core.ready(function () {
```

Comment:
Can this be DRY-ed along with `_simple_pie.html.erb` and `_stacked_area.html.erb`?